.gitignore

```diff
@@ -2,6 +2,7 @@
 __pycache__/
 *.py[cod]
 *$py.class
+test.junit.xml
```

Comment:
It's the junior version of the results that our internal build system needs for test runs